### PR TITLE
add community integration (fiber)

### PR DESCRIPTION
**Pull request by mstrYoda**

### CODE REVIEW

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -146,6 +146,21 @@ integrations:
...
+    description: |
+      Fiber is an Express inspired web framework built on top of Fasthttp, the fastest HTTP engine for Go.
+      Designed to ease things up for fast development with zero memory allocation and performance in mind.
+      With Open Policy Agent integration, you can run your Rego policies as part of the request lifecycle in the middleware.
+    labels:
+      category: application
+      layer: library
+    software:
+      - golang
+      - fiber
```
Comment (anderseknert) [inline docs/website/data/integrations.yaml:160]: You'll need to add the corresponding items to the "software" object further down in this file.

**Review by anderseknert — COMMENTED** [review]
Looks good! Just a small addition needed and then we can have this merged. Thanks! 👍

**Comment by anderseknert** [comment]
Oh, and you'll need to add a signoff to your commit as well. So, once you've fixed the software section with your additions, squash your commits into one, and then you can do:

```
git commit --ammend --signoff
git push --force
```

File: docs/website/data/integrations.yaml (hunk 1 above)
Comment (mstrYoda) [reply docs/website/data/integrations.yaml:160]: Oh, sorry I missed that part 🙏

**Review by anderseknert — APPROVED** [review]
LGTM. Thanks! 👏

**Pull request merged**